packages/code-du-travail-frontend/src/common/Answer.js

```diff
@@ -84,29 +109,62 @@ function Answer({
             title={title}
           />
         </StyledContent>
-        {linkedResults.matches.length > 0 && (
-          <StyledMenu>
-            <StyledMenuList>
-              <StyledMenuTitle>
-                Les articles pouvant vous interesser :
-              </StyledMenuTitle>
-              <ul>
-                {linkedResults.matches
-                  .filter(link => link.title !== title)
-                  .slice(0, 3)
-                  .map(link => (
-                    <StyledLi key={link.slug}>
-                      <Link
-                        href={`/${getRouteBySource(link.source)}/[slug]`}
-                        as={`/${getRouteBySource(link.source)}/${link.slug}`}
-                      >
-                        <a>{link.title}</a>
-                      </Link>
-                    </StyledLi>
-                  ))}
-              </ul>
-            </StyledMenuList>
-          </StyledMenu>
+        {relatedItems.length > 0 && (
+          <RelatedItems>
+            <StyledRelatedItemsTitle>
+              Autres contenus pouvant vous intéresser&nbsp;:
+            </StyledRelatedItemsTitle>
+            <List>
+              {relatedArticles
+                .filter(link => link.title !== title)
+                .slice(0, 3)
+                .map(link => (
+                  <StyledListItem key={link.slug}>
+                    <Link
+                      href={`/${getRouteBySource(link.source)}/[slug]`}
+                      as={`/${getRouteBySource(link.source)}/${link.slug}`}
+                      passHref
+                    >
+                      <LargeLink variant="light">
```

Comment:
Can we keept the previous style for Article and add Tile block under ?

Reply:
I followed the specs given by clement concerning the order but I'm going to make some proposals and see what we decide